gradle.properties

```diff
@@ -22,6 +22,6 @@ kotlin.code.style=official
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
 
-VERSION_NAME=3.1.0
+VERSION_NAME=3.2.0
 VERSION_CODE=1
-NEVIS_MOBILE_AUTHENTICATION_SDK_VERSION=3.1.0.1351
\ No newline at end of file
+NEVIS_MOBILE_AUTHENTICATION_SDK_VERSION=3.2.0.1382
```

Comment:
Please use `NEVIS_MOBILE_AUTHENTICATION_SDK_VERSION=3.2.+` as in coroutines app.
Sorry, that was wrong originally here.